package.json

```diff
@@ -41,7 +41,7 @@
     "travis:build": "NODE_ENV=production webpack --config config/prod.webpack.config.js",
     "travis:verify": "npm-run-all travis:build lint test",
     "verify": "npm-run-all build lint test",
-    "nightly": "npm run travis:verify"
+    "nightly": "npm-run-all build lint && npm run test -- -u"
```

Comment:
This should be only `npm-run-all build lint && npm run test` not needed to update snapshots while running tests.

Reply:
If we don't update the tests, travis will fail the build and won't push it out to the nightly environment.

Reply:
Oooh, that's nightly. Yeah, that is good to go.